dozer-types/src/models/connection.rs

```diff
@@ -226,3 +226,21 @@ pub enum ConnectionConfig {
     /// In yaml, present as tag" `!Dozer`
     Dozer(NestedDozerConfig),
 }
+
+impl ConnectionConfig {
+    pub fn get_type_name(&self) -> String {
```

Comment:
Can you help change [this snippet](https://github.com/getdozer/dozer/blob/main/dozer-cli/src/simple/build/contract/mod.rs#L110) to a call of this function?

Reply:
I would also use lower case for matching so we don't have to worry about case in various places.